wp-content/themes/pawar2018/js/scripts/app.js

```diff
@@ -17,6 +17,13 @@
   for (i = 0; i < accords.length; ++i) {
     let accordContent = accords[i].querySelector(".accordion-content");
     let accordTitle = accords[i].querySelector(".accordion-title");
+    if (window.location.hash) {
+      var hash = window.location.hash.substring(1);
+      if (hash === accordTitle.innerText.toLowerCase().split(' ').join('-')) {
```

Comment:
If all `.accordionTitle`'s have a `name` attribute (which seems to match what you're looking for), would be a little cleaner to check `hash` against `accordionTitle.getAttribute('name')`. (Would avoid the `toLowerCase`, `split`, and `join` operations.)